book/caveats.nim

```diff
@@ -0,0 +1,81 @@
+import nimib, nimibook
+
+
+nbInit
+nbUseNimibook
+
+nbText: """
+# Caveats
+There are some caveats when working with norm that you need to consider and strategies to work around them.
```

Comment:
Please use consistent naming: Norm instead of norm.

It just looks tidier this way.

Reply:
Done